extensions/wikia/ImageReview/ImageReviewEventsHooks.class.php

```diff
@@ -1,11 +1,19 @@
 <?php
 
-use \Wikia\Logger\WikiaLogger;
-use \Wikia\Tasks\Tasks\ImageReviewTask;
+use Wikia\Logger\WikiaLogger;
+use Wikia\Rabbit\ConnectionBase;
+use Wikia\Tasks\Tasks\ImageReviewTask;
 
 class ImageReviewEventsHooks {
 	public static function onUploadComplete( UploadBase $form ) {
-		static::createAddTask( $form->getTitle() );
+		global $wgCityId, $wgImageReviewTestCommunities;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$wgCityId" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$wgImageReviewTestCommunities" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)